README.md

```diff
@@ -1,41 +1,51 @@
 
-1. Install [Python](https://www.python.org/downloads/), [Pipenv](https://docs.pipenv.org/) and [Postgres](https://www.postgresql.org/) on your machine, if you do not have them already
+ 1. Install [Python](https://www.python.org/downloads/), [Pipenv](https://docs.pipenv.org/) and [Postgres](https://www.postgresql.org/) on your machine, if you do not have them already
   * To install Pipenv on a Mac, you can run `$ brew install pipenv`
-2. You should have Python 3.8.5
+ 2. You should have Python 3.8.5
   ```
   $ python3 --version
   => Python 3.8.5
   ```
-3. You should have Postgres
+ 3. You should have Postgres
   ```
   $ which psql
   => /Applications/Postgres.app/Contents/Versions/latest/bin/psql
   ```
-4. You should have pipenv
+ 4. You should have pipenv
   ```
   $ pipenv --version
   => pipenv, version 2020.8.13
   ```
-5. Fork and clone down the repository
-6. Change into the directory `$ cd eth_crowdfund_be/eth_crowdfund_api`
-7. You will need to work in a virtual environment. Why? Using a virtual environment for Python projects allows us to have an isolated working copy of Python so we can work on a specific project without worrying about affecting other projects.
-8. Within the `/eth_crowdfund_api` directory:  
- a. Run `# pipenv --three` to create the virtual environment  
- b. Run `$ pipenv shell` to activate the project virtual environment. When you are done working on the project, you should execute `$ exit` to exit the virtual environment  
- c. Run `$ pipenv install flask flask-sqlalchemy psycopg2 flask-migrate flask-script marshmallow flask-bcrypt pyjwt` to install all dependencies  
- d. Run `$ createdb eth_crowdfund_api_db` to create the app database  
- e. Run
- ```
- $ export FLASK_ENV=development  
- $ export JWT_SECRET_KEY=hhgaghhgsdhdhdd
- ```
- to set the system environment variables.
- You should now be able to run the app:  
- 9. Run
+ 5. Fork and clone down the repository
+ 6. Change into the directory `$ cd eth_crowdfund_be`
+ 7. You will need to work in a virtual environment. Why? Using a virtual environment for Python projects allows us to have an isolated working copy of Python so we can work on a specific project without worrying about affecting other projects.
+ 8. Within the root directory:  
+  a. Run `# pipenv --three` to create the virtual environment  
+  c. Run `$ pipenv install` to install all dependencies  
+  d. Run `$ createdb eth_crowdfund_api_db` to create the app database  
+  e. Run `$ createdb eth_crowdfund_api_db_test` to create the testing database  
+  f. Run `$ touch .env` to create an enviornment file 
+  e. Within the `.env` file add appropriate environment values for flask enviornment, database URLs, and localhost port
+  ```
+   FLASK_ENV=development
+   SQLALCHEMY_DATABASE_URI="postgresql://postgres:password@localhost/eth_crowdfund_api_db"
+   SQLALCHEMY_TEST_DATABASE_URI="postgresql://postgres:password@localhost/eth_crowdfund_api_db_test"
+   PORT=3000
```

Comment:
Is this sensitive information? Why do we even have a .env file....

Reply:
That's a good question. I figured it was okay because it's required for the app to run and I don't think anyone can abuse it. If we start adding API keys we should definitely keep them secret, but just mention they are needed in the .env file.